src/pytestqt/qml/botloader.qml

```diff
@@ -0,0 +1,18 @@
+import QtQuick 2.15
```

Comment:
Seems this file needs to be included as a data file in `setup.py`.

Reply:
yes, will do.